ehsm-service/napiparams.js

```diff
@@ -0,0 +1,119 @@
+const keyspec= {
+  type : 'const',
+  arr: ['EHM_AES_GCM_128','EHM_RSA_3072'],
+  errortext: "keyspec is must 'EHM_AES_GCM_128' or 'EHM_RSA_3072'",
+};
+const pramas = {
+  CreateKey: {
+    keyspec,
+    origin: {
+      type : 'const',
+      arr: ['EHO_INTERNAL_KEY','EHO_EXTERNAL_KEY'],
+      errortext: "keyspec is must 'EHO_INTERNAL_KEY' or 'EHO_EXTERNAL_KEY'",
+    }
+  },
+  Encrypt: {
+    cmk_base64: {
+      type: 'string',
+      maxLength: 128,
+      length: 100
+    },
+    plaintext: {
+      type: 'string',
+      maxLength: 128,
```

Comment:
here, the plaintext can support len<6K, not 128

Reply:
The maxLength will be change to 6*1024.